app/models/notification.rb

```diff
@@ -1,9 +1,15 @@
 class Notification < ApplicationRecord
   belongs_to :user
 
+  @@NOTIFICATION_TYPES = {LIKES: "likes",COMMENTS:"comments",FOLLOW:"follow",UNFOLLOW:"unfollow",ACCEPTANCE:"follow_acceptance"}
+
+  # getter
+  def self.NOTIFICATION_TYPES
+    @@NOTIFICATION_TYPES
+  end
   
   def self.create_notification(parameters)
-    @notification=Notification.new(user_id:parameters[:user_id],actor_id:parameters[:actor_id],notification_type:parameters[:notification_type],target_type:parameters[:target_type],target_id:parameters[:target_id])
-    @notification.save
+    notification=Notification.new(user_id:parameters[:user_id],actor_id:parameters[:actor_id],notification_type:parameters[:notification_type],target_type:parameters[:target_type],target_id:parameters[:target_id])
```

Comment:
Since the parameters has key are same as in Model object. I think You can just simply do this.
```
notification = Notification.new(parameters)
```
Please check and verify this.

Reply:
notification = Notification.new(parameters)
Done